### PR TITLE
Update ClubPage.module.css

### PR DESCRIPTION
**Pull request by brachiotusk**
I just wanted to some simple styling

### CODE REVIEW

**Review by FireStreaker2 — CHANGES_REQUESTED** [review]
you can't use pure selectors in css modules, you should probably add ``.infoTable`` before the pure selectors

File: src/styles/ClubPage.module.css [hunk 1]
```diff
@@ -8,4 +8,27 @@
...
+}
+
+.infoTable {
+	text-align: center;
+	background-color: #68a7ca;
+	padding: 20px;
+	border-radius: 10px;
+}
+
+table {
```
Comment (FireStreaker2) [inline src/styles/ClubPage.module.css]:
```suggestion
.infoTable table {
```

Comment (FireStreaker2) [reply src/styles/ClubPage.module.css]: ^ like this

**Comment by brachiotusk** [comment]
oh my bad , ill fix that

File: src/styles/ClubPage.module.css [hunk 2]
```diff
@@ -17,18 +17,18 @@
...
 	margin: 0 auto;
 }
 
-table, th, td {
+.infoTable table, th, td {
 	border: 1px solid #ddd;
 }
 
-th, td {
+.infoTable th, td {
```
Comment (FireStreaker2) [inline src/styles/ClubPage.module.css]:
```suggestion
.infoTable table, .infoTable th, .infoTable td {
  border: 1px solid #ddd;
}

.infoTable th, .infoTable td {
```
make sure you include these too

**Comment by FireStreaker2** [comment]
make sure you include the other ones too

**Review by FireStreaker2 — APPROVED** [review]
looks good now

**Pull request merged**